modules/rn-ocp-release-notes-api.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: CONCEPT
+[id="rn-ocp-release-notes-api_{context}"]
+= API server
+
+[role=”_abstract”]
+Extended loopback certificate validity to three years for kube-apiserver::
+* Before this update, the self-signed loopback certificate for the Kubernetes API Server expired after one year. With this release, the expiration date of the certificate is extended to three years.
```

Comment:
Since the preview is not loading, I am not sure, but I think we don't want bullets in DL items unless we need a bulleted list inside a DL
```suggestion
Extended loopback certificate validity to three years for kube-apiserver::
Before this update, the self-signed loopback certificate for the Kubernetes API Server expired after one year. With this release, the expiration date of the certificate is extended to three years.
```